websites/M/marcuscodes.me/presence.ts

```diff
@@ -3,13 +3,14 @@ const presence = new Presence({
 });
 
 const timeStamp = Math.floor(Date.now() / 1000);
+const page = window.location.href;
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "circlegrad",
     startTimestamp: timeStamp
   };
-  const page = window.location.href;
+
   if (page == "https://marcuscodes.me/") {
```

Comment:
double check if it still works with this change

Reply:
yes